src/app/pages/allow-diagnostics/allow-diagnostics.tsx

```diff
@@ -21,8 +21,8 @@ export const AllowDiagnosticsPage = () => {
       if (typeof areDiagnosticsAllowed === undefined) return;
       setHasAllowedDiagnostics(areDiagnosticsAllowed);
       if (areDiagnosticsAllowed) {
-        initSentry();
-        void initSegment();
+        SENTRY_ENABLED && initSentry();
+        SEGMENT_ENABLED && void initSegment();
```

Comment:
I'd move these checks into the functions themselves. Consider that we might call these in multiple places, so to avoid repeating the conditional in each place, better to put inside.

Reply:
Yeah, def overkill here but was just killing it before it does the log again.